src/icon/icon.jsx

```diff
@@ -4,7 +4,7 @@ import classnames from 'classnames'
 import { onlyUpdateForPropTypes } from 'recompose'
 import styles from '../../src/icon/icon.styles.scss'
 
-const labelFactory = htmlFor => label => <label htmlFor={htmlFor}>{label}</label>
+const labelFactory = code => label => <span id={code} aria-hidden="true">{label}</span>
```

Comment:
updated it because `<label>` is only allow to label an input and `for` attribute works only to target input